### PR TITLE
TSK-774: set Flags for listValues()

### PR DESCRIPTION
**Pull request by BVier**
Removed callback_info and custom_attributes because DB2 can't ORDER BY an CLOB value and they aren't necessary.

### CODE REVIEW

File: lib/taskana-core/src/main/java/pro/taskana/mappings/QueryMapper.java [hunk 1]
```diff
@@ -672,8 +672,8 @@
         + "group by WORKBASKET_ID ) AS f where max_read = 1 ) "
         + "</if> "
         + "<if test='taskIds != null'>AND t.ID IN(<foreach item='item' collection='taskIds' separator=',' >#{item}</foreach>)</if> "
-        + "<if test='externalIdIn != null'>AND a.EXTERNAL_ID IN(<foreach item='item' collection='externalIdIn' separator=',' >#{item}</foreach>)</if> "
-        + "<if test='externalIdLike != null'>AND (<foreach item='item' collection='externalIdLike' separator=' OR '>UPPER(a.EXTERNAL_ID) LIKE #{item}</foreach>)</if> "
+        + "<if test='externalIdIn != null'>AND t.EXTERNAL_ID IN(<foreach item='item' collection='externalIdIn' separator=',' >#{item}</foreach>)</if> "
```
Comment (BVier) [inline lib/taskana-core/src/main/java/pro/taskana/mappings/QueryMapper.java]: Changed this because attachment doesn't have such an attribute. So it's probably a typo.
In my opinion it wouldn't make sense to do an own pull request for this.

File: lib/taskana-core/src/main/java/pro/taskana/impl/TaskQueryImpl.java [hunk 2]
```diff
@@ -1014,14 +1014,16 @@ private void setupAccessIds() {
...
             checkOpenAndReadPermissionForSpecifiedWorkbaskets();
             setupAccessIds();
+
             if (columnName.equals(TaskQueryColumnName.CLASSIFICATION_NAME)) {
                 joinWithClassifications = true;
-                addClassificationNameToSelectClauseForOrdering = true;
             }
-            if (columnName.equals(TaskQueryColumnName.A_CLASSIFICATION_NAME)) {
-                joinWithAttachments = true;
+            List<TaskQueryColumnName> attachmentColumns = Arrays.asList(
```
Comment (TitusLabs) [inline lib/taskana-core/src/main/java/pro/taskana/impl/TaskQueryImpl.java]: I would prefer it to select them by convention like all columns starting with A_
You will not adapt this while adding a new column to the enum.
furthermore this is a static value which could be extracted. I would actually include it in the enum class itself.

Comment (BVier) [reply lib/taskana-core/src/main/java/pro/taskana/impl/TaskQueryImpl.java]: Good point!
Also didn't liked my solution at this point, but I wasn't aware that I could add such a function to a Enum class.
Thanks for reporting!

**Comment by BerndBreier** [comment]
This pull request is replaced by PR "B vier tsk 772 lazy #641" which fixes problems detected during travis build / test of this PR.
